core/templates/dev/head/domain/skill/EditableSkillBackendApiService.ts

```diff
@@ -50,6 +50,24 @@ oppia.factory('EditableSkillBackendApiService', [
       });
     };
 
+    var _fetchMultiSkills = function(skillIds, successCallback, errorCallback) {
+      var skillDataUrl = UrlInterpolationService.interpolateUrl(
+        SKILL_DATA_URL_TEMPLATE, {
+          skill_ids: skillIds.join(',')
```

Comment:
ditto, key name should be comma_separated_skill_ids

Reply:
Done.